requery-processor/src/main/java/io/requery/processor/EntityGenerator.java

```diff
@@ -34,13 +34,7 @@
 import io.requery.proxy.PropertyState;
 
 import javax.annotation.processing.ProcessingEnvironment;
-import javax.lang.model.element.Element;
-import javax.lang.model.element.ElementKind;
-import javax.lang.model.element.ExecutableElement;
-import javax.lang.model.element.Modifier;
-import javax.lang.model.element.PackageElement;
-import javax.lang.model.element.TypeElement;
-import javax.lang.model.element.VariableElement;
```

Comment:
Oops, sorry for that. I need to relax my IntelliJ settings so it won't do this automatically. Thanks for merging so quick!